src/display/editor/drawers/signaturedraw.js

```diff
@@ -491,32 +576,55 @@ class SignatureExtractor {
     );
 
     const threshold = this.#guessThreshold(histogram);
-    const contourList = this.#findContours(
-      uint8Filtered,
-      width,
-      height,
-      threshold
+    const contourList = this.#findContours(buffer, width, height, threshold);
+
+    return this.processDrawnLines(
+      { curves: contourList, width, height },
+      pageWidth,
+      pageHeight,
+      rotation,
+      innerMargin,
+      true,
+      true
     );
-    const linesAndPoints = [];
+  }
 
+  static processDrawnLines(
+    lines,
+    pageWidth,
+    pageHeight,
+    rotation,
+    innerMargin,
+    mustSmooth,
+    areContours
+  ) {
     if (rotation % 180 !== 0) {
       [pageWidth, pageHeight] = [pageHeight, pageWidth];
     }
 
-    // The points need to be converted into page coordinates.
-    const ratio = 0.5 * Math.min(pageWidth / width, pageHeight / height);
+    const { curves, thickness, width, height } = lines;
```

Comment:
It doesn't seem that any of the call-sites include `thickness`?

Reply:
It's to handle the case where the user drew a signature with a pointer in the dialog.
So it isn't used now but it will in a following patch.